pyproject.toml

```diff
@@ -18,7 +18,7 @@ classifiers = [
 ]
 requires-python = ">=3.7"
 dependencies = [
-    "Flask",
+    "Flask[async]",
```

Comment:
Don't make the `async` extra a requirement. `ensure_sync` will fail with an error message explaining the extra once a project actually uses async views.